ds3-autogen-c/src/test/java/com/spectralogic/ds3autogen/c/CCodeGeneratorSpectraS3Requests_Test.java

```diff
@@ -46,7 +43,7 @@
 public class CCodeGeneratorSpectraS3Requests_Test {
 
     @Test
-    public void testGenerateSpectraS3DeleteBucketRequest() throws IOException, ParserException, ResponseTypeNotFoundException, TypeRenamingConflictException, ParseException, TemplateModelException {
+    public void testGenerateSpectraS3DeleteBucketRequest() throws IOException, ParseException, TemplateModelException {
```

Comment:
Nm. ParserException was changed to runtime, but ParseException is not one of ours.